src/App/ComponentsDocumentation/components/InputField/index.js

```diff
@@ -2,15 +2,15 @@ import React, { forwardRef, useRef } from "react";
 import { ComponentPreview, DocContainer } from "@docutils";
 import CodeTags from "@components/CodeTags";
 import InputGroup from "@components/InputGroup";
-import { inputShowCase, textareaShowCase } from "./constants";
+import { inputShowCase, textareaShowCase, contentGuidelines } from "./constants";
 
 const Overview = ({ scrollToRef }) => (
     <section id="input-field-overview">
         <h2 id="input-field">Input field</h2>
         <p>Use a Input field when the expected user input is a single line of text. A Input field must always have a Input label that clearly describes the type of input a field requires and it may also have a Placeholder example text, refer to <a href="javascript:;" onClick={() => scrollToRef()}>content guidelines</a> for more information.</p>
```

Comment:
I think we need to rewrite this text... a/an typo and loooooong ass sentence

Reply:
Agree